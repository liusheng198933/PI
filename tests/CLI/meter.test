#!/bin/sh
./run_one_test.py $srcdir/testdata meter stats.json
