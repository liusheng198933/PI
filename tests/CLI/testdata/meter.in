meter_read_spec MeterA 9
meter_set MeterA 9 1:3 2:12
meter_read_spec MeterA 9
meter_set MeterA 9 400:300 1000:300
meter_read_spec MeterA 9
