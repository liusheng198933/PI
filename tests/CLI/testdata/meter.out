????
Device assigned successfully.
Selecting device.
????
Error when trying to read meter spec
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 3
	Peak information rate: 2
	Peak burst size: 12
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 1
	Committed burst size: 3
	Peak information rate: 2
	Peak burst size: 12
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 400
	Committed burst size: 300
	Peak information rate: 1000
	Peak burst size: 300
????
Dumping meter spec:
	Meter unit: bytes
	Meter type: color unaware
	Committed information rate: 400
	Committed burst size: 300
	Peak information rate: 1000
	Peak burst size: 300
????
